src/main/java/frc/robot/subsystems/leds/Leds.java

```diff
@@ -3,43 +3,56 @@
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot.subsystems.leds;
-
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
 import java.util.Optional;
-
 import com.ctre.phoenix.led.CANdle;
 import com.ctre.phoenix.led.TwinkleAnimation;
 
-import edu.wpi.first.wpilibj.DriverStation;
-import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
+
 public class Leds extends SubsystemBase {
+
+  Optional<Alliance> alliance;
+  private TwinkleAnimation twinkleblueAnimation;
+  private TwinkleAnimation twinkleredAnimation;
+
   private CANdle candle1;
-  TwinkleAnimation twinkleblueAnimation = new TwinkleAnimation(0, 0, 255);
-  TwinkleAnimation twinkleredAnimation = new TwinkleAnimation(255, 0, 0);
-  TwinkleAnimation twinklewhiteAnimation = new TwinkleAnimation(255, 255, 255);
-  /** Creates a new Leds. */
+
+  /** Creates a new CANdle */
   public Leds() {
+    setupLeds();
+  }
+
+  private void setupLeds() {
+    twinkleblueAnimation = new TwinkleAnimation(0, 0, 255);
+    twinkleredAnimation = new TwinkleAnimation(255, 0, 0);
+    candle1 = new CANdle(1);
+    
+    alliance = DriverStation.getAlliance();
+
+    setLedsUsingAllianceColor();
+
   }
 
   @Override
   public void periodic() {
+    // This method will be called once per scheduler run
+  }
 
-    Optional<Alliance> ally = DriverStation.getAlliance();
-    if (ally.isPresent()) {
-      if (ally.get() == Alliance.Red) {
-        // Put what you want it to do here.
+  public void setLedsUsingAllianceColor() {
+
+    if (alliance.isPresent()) {
+      if (alliance.get() == Alliance.Red) {
         candle1.animate(twinkleredAnimation);
+
       }
-      if (ally.get() == Alliance.Blue) {
+      if (alliance.get() == Alliance.Blue) {
         // Put what you want it to do here.
         candle1.animate(twinkleblueAnimation);
       }
-      else {
-        // Put what you want it to do here.
-        candle1.animate(twinklewhiteAnimation);
-      }
     }
-    // This method will be called once per scheduler run
   }
 }
```

Comment:
You'll want to add another function here that sets the LEDs to strobe, so what we can make the LEDs strobe from outside of this subsystem